certified-connectors/ActiveCampaign/apiProperties.json

```diff
@@ -0,0 +1,43 @@
+{
+  "properties": {
+    "capabilities": [],
+    "connectionParameters": {
+      "apiKey": {
+        "type": "securestring",
+        "uiDefinition": {
+          "constraints": {
+            "clearText": false,
+            "required": "true",
+            "tabIndex": 2
+          },
+          "description": "The KEY for this API",
+          "displayName": "KEY",
+          "tooltip": "Provide your KEY"
+        }
+      },
+      "hostName": {
+        "type": "string",
+        "uiDefinition": {
+          "constraints": {
+            "required": "true"
+          },
+          "description": "ActiveCampaign Hostname (Example: yourAccountName.api-us1.com).",
+          "displayName": "ActiveCampaign REST Endpoint Hostname",
+          "tooltip": "Provide the Hostname"
+        }
+      }
+    },
+    "iconBrandColor": "#356AE6",
+    "publisher": "Microsoft",
```

Comment:
@mmchome can you please confirm that you are working on behalf of Active Campaign and that you are not submitting this connector as an independent publisher?

Reply:
@mmchome can you please clarify?

Reply:
no I am working on behalf of Microsoft

Reply:
@mmchome I had a meeting with Philippe and he clarified that for me. Thank you so much,